.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,5 +6,7 @@
 - [ ] Added tests that cover your change (if possible)
 - [ ] Added/modified documentation as required (such as the `README.md`, and `examples` directory)
 - [ ] Manually tested
+- [ ] Added labels for change area and target version
```

Comment:
Perhaps add a couple of examples so they are easier to find by external contributors?

Reply:
Done. (In `Add example labels in PR checklist `.)